### DISCUSSION TITLE
How to make a mesh transparent.

### DISCUSSION THREAD

**Discussion by juanccilleruelo** [pyvista/pyvista#3958] [Q&A] [answered]
Hello all!
I'm new with vista and with python too!

I have a cube representing the whole dimensions of a furniture piece. Inside this cube, I want to create each board of the furniture piece. What I need is to set this main cube transparent in a percentage. 

**How can I set the color and the percentage of transparency of this main cube?**

I really appreciate any help you can provide.

**Comment by MatthewFlamm** [comment, marked as answer]
Try something like

```py
import pyvista as pv
pl = pv.Plotter()
pl.add_mesh(pv.Cube(), opacity=0.5)
# add in other meshes
pl.show()
```

**Comment by juanccilleruelo** [comment]
I understand that the opacity parameter is to modify how the mesh is added to the plotter, not a property of the own mesh created with the pv.Cube(). 

Thank you!

**Comment by ycebear** [comment]
Perhaps you prefer a wire grid: `pl.add_mesh(mesh, style='wireframe', color='k', line_width=2)`   [see here](https://docs.pyvista.org/version/stable/examples/02-plot/anti-aliasing#sphx-glr-examples-02-plot-anti-aliasing-py)
